src/__tests__/stateFromElement-test.js

```diff
@@ -9,23 +9,24 @@ import {
 import {convertToRaw} from 'draft-js';
 import {jsdom} from 'jsdom';
 import fs from 'fs';
+import {EOL} from 'os';
 import {join} from 'path';
 
 let document = jsdom('<!doctype html><html><body></body></html>');
 
 // This separates the test cases in `data/test-cases.txt`.
-const SEP = '\n\n#';
+const SEP = `${EOL}${EOL}#`;
```

Comment:
Good point. I recommend instead to add `test/test-cases.txt text eol=lf` to `.gitattributes`. It will tell the git client to keep LF line endings on checkout.